README.md

```diff
@@ -1,36 +1,114 @@
+
+With `requirements` you only need to install one lightweight package and write one human-readable file detailing your dependencies, then a single function call ensures you have all your dependencies available.
+
+### In-line installation
 
 ```
 if(!require(mgsub)) install.packages('mgsub')
 ```
 
-This checks if a package is already installed and if not, installs it. Of course, this example fails to then load the package so the code will fail. But this always installs the latest version available for the person running the code. This can be an issue because the packages your code depends on may have made breaking changes. Now you've installed an incompatible version.
+Often code may start with a line that checks if a package is already available and if not, install it. Of course, this example fails to load the package after installation so a user may still have issues running a file. There are good arguments that analysis code should not be in the business of installing packages which might affect other projects, so code like this could be considered undesirable.
+
+The other issue here is that this always installs the latest version available for the person running the code. This can be an issue because the packages your code depends on may have made breaking changes. Now you've installed an incompatible version.
+
+### `pacman`
+
+`pacman` provides a new command, `pacman::p_load()` which will install a package if it's not available and then load it. However, it does not allow you to specify specific versions, so it's not really a solution to managing requirements.
+
+### `packrat`
 
-### packrat
+`packrat` is a great way of bundling code and packages together. It allows you to have multiple versions of packages installed across different projects. `requirements` is 100% compatible with `packrat`, if you're using `packrat` it will check against project specific versions and install into the `packrat` library. This allows you to have a standard set of packages that you use, including versions, which can initialize new projects without multiple installation commands. It also frees you from the complexities of sharing `packrat` environments between different people and provides a more human readable list of project dependencies.
 
-packrat is a great way of bundling code and packages together. It allows you to have multiple versions of packages installed across different projects. `requirements` is 100% compatible with packrat, if you're using packrat it will check against project specifc versions and install into the packrat library. This allows you to have a standard set of packages that you use, including versions, which can initialize new projects without multiple installation commands.
+## Why `rip_*`?
 
-### pacman
+`pip` stand for 'Python Installs Packages' and so we wanted to call the project `rip` for 'R Installs Packages'. Sadly, that name is already in use on CRAN, so we satisfied ourselves by leveraging `rip` in our function names.
```

Comment:
`pip` is actually a recursive acronym standing for 'pip installs packages'  This might break the parallel you were trying to make between the definitions of `pip`/`rip` so not sure what edit would be best (why i didnt just edit and commit)

I also lean more towards `rip` standing for 'requirements installs packages', but it could just be a fluid acronym to be interpreted by the reader.

Reply:
what do you think about the updated section?